test/ghc.jl

```diff
@@ -0,0 +1,21 @@
+@testset "GHC" begin
+    setify(lists) = Set(Set.(lists))
+  
+    𝒮 = georef((z=[ones(10,10) 2ones(10,10); 3ones(10,10) 4ones(10,10)],))
+    p = partition(𝒮, GHC(4, 1.0))
+    @test length(p) == 4
+    @test all(nelements.(p) .== 100)
+    p1, p2, p3, p4 = p
+    @test mean(coordinates(centroid(p1, ind)) for ind in 1:nelements(p1)) == [5.0,5.0]
+    @test mean(coordinates(centroid(p2, ind)) for ind in 1:nelements(p2)) == [15.0,5.0]
+    @test mean(coordinates(centroid(p3, ind)) for ind in 1:nelements(p3)) == [5.0,15.0]
+    @test mean(coordinates(centroid(p4, ind)) for ind in 1:nelements(p4)) == [15.0,15.0]
+  
+    𝒮 = georef((z=[√(i^2+j^2) for i in 1:100, j in 1:100],))
+    p = partition(𝒮, GHC(50, 1.0))
+    @test length(p) == 49
+  
+    if visualtests
+      @test_reference "data/ghc.png" plot(p)
+    end
```

Comment:
I see that you are copying/pasting the tests from SLIC. That is not needed at this point. We will need specific tests for GHC when its implementation is ready. I would just erase this from the PR to avoid reading code that is not working yet.

Reply:
Ok, that sounds good